main.go

```diff
@@ -145,6 +146,11 @@ func main() {
 		Port:               9443,
 		LeaderElection:     enableLeaderElection,
 		LeaderElectionID:   "operator-lock",
+		// We should test this again in the future to see if the problem with LicenseKey updates
```

Comment:
Is there a link to a GitHub issue that we can include here? This comment is not going to be useful in 6 months time without a reference.

Reply:
I added more to the comment and also a link to the controller-runtime issue.